Modules/Core/tests/Feature/SessionsControllerTest.php

```
@@ -105,6 +108,305 @@ public function it_displays_password_reset_form_with_token(): void
+
+        $this->assertEquals(2, RateLimiter::attempts($throttleKey));
+
+        /** Act */
+        $this->post(route('sessions.authenticate'), [
+            'email' => 'test@example.com',
+            'password' => $password,
+        ]);
+
+        /* Assert */
+        $this->assertEquals(0, RateLimiter::attempts($throttleKey));
+    }
```

PHPMD (2.15.0)

The method it_clears_rate_limiter_on_successful_authentication is not named in camelCase. (undefined)

(CamelCaseMethodName)